util/test/chpl_launchcmd.py

```diff
@@ -307,6 +308,13 @@ def job_status(job_id, output_file):
                 raise ValueError('[Error: output file from job (id: {0}) does not exist at: {1}]'.format(
                     job_id, output_file))
 
+            # try removing the file stdin was copied to, might not exist
+            logging.debug('removing stdin file.'
```

Comment:
huh, good catch. I wonder how I managed to delete that between testing and committing